src/services/types.ts

```diff
@@ -214,6 +214,14 @@ namespace ts {
         installPackage?(options: InstallPackageOptions): Promise<ApplyCodeActionCommandResult>;
     }
 
+    export interface Options {
+        readonly quote?: "double" | "single";
+        readonly includeCompletionsForExternalModuleExports?: boolean;

```

Comment:
`includeCompletionsForModuleExports` because nobody after 2015 should ever use the term "external module"